aml-development/frontend
# Fix(tour): fix tour routing on ci-latest

Should fix routing issues with ci.
How to test:
pull the code, and make sure tour continues to work as expected.
Once merged, will deploy on ci to see if code functions as expected.
